src/components/Search/Search.jsx

```diff
@@ -45,10 +45,11 @@ function Search() {
     <div>
       {searchResults.map((giphy) => {
         return ( 
-        <img src={giphy.images.original.url} 
+        <img src={giphy.images.fixed_height.url} 
         key={giphy.id} />
         )
       })}
+      <button onClick={addFavorite}>Favorite this!</button>
```

Comment:
I think what we will want is:

<button onClick={() => addFavorite(giphy)}>Favorite this!</button>